src/Homepage.js

```diff
@@ -1,23 +1,21 @@
-import React, {Component} from 'react';
+import React from 'react';
 
-class Homepage extends Component { 
+function Homepage(props) { 
 
-    constructor(props) 
-    { 
-        super(props); 
+    const SwipePage = () => {
+        //Swipe
     }
     
-    render() {
+    return(
 
-        return(
         <article className="homepage">
-            <title>{this.props.title}</title>
-            <h3>{this.props.subtitle}</h3>
-            <img src={this.props.image} />
-            <par>{this.props.description}</par>
+            <title>{props.title}</title>
+            <h3>{props.subtitle}</h3>
+            <img src={props.image} />
+            <par>{props.description}</par>
```

Comment:
What is `<par>` meant to be? 🤔 

![image](https://user-images.githubusercontent.com/3296643/98467136-0d02c200-21d4-11eb-8cf0-f810692f1acd.png)